app/src/main/java/com/androidavanzado/covid19stadistics/util/MonthToNameMonth.kt

```diff
@@ -0,0 +1,25 @@
+package com.androidavanzado.covid19stadistics.util
+
+class MonthToNameMonth {
+
+    operator fun invoke(month: String) : String {
+
+        val nameMonth = when (month) {
```

Comment:
You could use the date formats and not have a when to know the name of the month.

Reply:
done